.github/kokoro/kokoro-cfg.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
```

Comment:
Let's add a comment for what purpose this script serves. Looks to generate the continuous and presubmit configs from a template? A helper/developer script?